component/common/src/main/java/org/exoplatform/management/common/AbstractOperationHandler.java

```diff
@@ -148,7 +148,8 @@ protected final Identity getIdentity(String id) {
         return spaceIdentity;
       }
     } catch (Exception e) {
-      log.error("Error while retrieving identity: ", e);
+      // issue : We got a log error with a stack trace of the exception.
+      log.warn("Failed to retrieve identity from id : "+id);

```

Comment:
In some cases we need to export questions of an owner that is no longer exist ==> When exporting for this non-existent user, we got a log error with a stack trace of the exception however this not an error in fact we know that the user is not exist.